lib/parsers/ccda/document.js

```diff
@@ -61,6 +71,13 @@ Parsers.CCDA.document = function (ccda) {
       }
     },
     documentation_of: documentation_of_list,
+    encounterParticipant: {
+      name: performer_name_dict,
+      phone: {
+        work: performer_phone
+      },
+      address: performer_addr
+    },
```

Comment:
@anmolagar You're parsing data from `encompassingEncounter/encounterParticipant` but you're not actually using any of those parsed fields – this just outputs the data that's coming out of `documentationOf/performer` and is currently output in `documentation_of`.

It looks like very few of the samples are using the `encounterParticipant` field. Are you running into samples where this is important data?